ush/setup.py

```diff
@@ -88,8 +91,8 @@ def load_config_for_setup(ushdir, default_config, user_config):
         raise Exception(errmsg)
 
     # Make sure the keys in user config match those in the default
-    # config.
-    invalid = check_structure_dict(cfg_u, cfg_d)
+    # config. Skipping during uwtools integration activities.
+    invalid = {}
```

Comment:
This should likely be removed altogether in favor of a SRW Schema at some point in the future.